### PR TITLE
Add AZ resources key vault and storage account

### PR DESCRIPTION
**Pull request by RajasekarVadivelu**

### CODE REVIEW

File: sandbox/rajasekar/key-vault.tf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+resource "azurerm_key_vault" "raj-key-vault" {
```
Comment (shantanuinamdar) [inline sandbox/rajasekar/key-vault.tf:1]: Suggestions from @sukhdeepsingh1 
resource naming, should be in snake_case & not to use type in the name

File: sandbox/rajasekar/main.tf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+provider "azurerm" {
+  subscription_id = "18c8d488-9b69-4619-ab58-724f279fe48e" // Sandbox
+  features {}
+}
+
+data "azurerm_client_config" "current" {
+}
+
+resource "azurerm_resource_group" "sandbox-rajasekar" {
```
Comment (shantanuinamdar) [inline sandbox/rajasekar/main.tf:39]: same as above

Comment (RajasekarVadivelu) [reply sandbox/rajasekar/main.tf]: Replaced

File: sandbox/rajasekar/storage-account.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+resource "azurerm_storage_account" "sandbox-storage-account" {
+  name                     = "sbstorageaccount"
+  resource_group_name      = azurerm_resource_group.sandbox_rg.name
+  location                 = azurerm_resource_group.sandbox_rg.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
```
Comment (sukhdeepsingh1) [inline sandbox/rajasekar/storage-account.tf:6]:
```suggestion
  account_replication_type = "LRS"
```

**Review by sukhdeepsingh1 — CHANGES_REQUESTED** [review]

File: sandbox/rajasekar/storage-account.tf (hunk 3 above)
Comment (RajasekarVadivelu) [reply sandbox/rajasekar/storage-account.tf]: changed to LRS

**Review by sukhdeepsingh1 — APPROVED** [review]

**Pull request merged**